parser-typechecker/src/Unison/Codebase/TranscriptParser.hs

```diff
@@ -88,8 +89,12 @@ instance Show Stanza where
 
 parseFile :: FilePath -> IO (Either Err [Stanza])
 parseFile filePath = do
-  txt <- readUtf8 filePath
-  pure $ parse filePath txt
+  exists <- doesFileExist filePath
+  if exists then do
+    txt <- readUtf8 filePath
+    pure $ parse filePath txt
+  else
+    pure $ Left $ show filePath ++ " does not exist"
```

Comment:
originally a failure due to file not found in `readUtf8` resulted in an exception